algebra/group/default.lean

```diff
@@ -484,23 +484,31 @@ section add_comm_group
 
 end add_comm_group
 
-variables {β : Type*} [group α] [group β] {a b : α}
+variables  [group α] {β : Type*} [group β]
 
 /-- Predicate for group homomorphism. -/
-def is_group_hom (f : α → β) : Prop :=
-∀ a b : α, f (a * b) = f a * f b
+class is_group_hom (f : α → β) :=
+  (hom_mul : ∀ a b : α, f (a * b) = f a * f b)
```

Comment:
Why did you make this a `Type`? We really want `is_*` things to be `Prop`s

Reply:
style: 0 indent

Reply:
Turned `is_group_hom` back into a `Prop` in https://github.com/leanprover/mathlib/pull/94/commits/ca775263f8158964cccd5495d0c6108ddba134ca, although this caused two typeclass inference problems I don't understand: see https://github.com/leanprover/mathlib/pull/94#pullrequestreview-110300976.

Style issues hopefully all fixed in https://github.com/leanprover/mathlib/pull/94/commits/91693ec5a4d0762766254c8a4d3153659349e094.